react/features/base/logging/JitsiMeetLogStorage.js

```diff
@@ -63,13 +63,13 @@ export default class JitsiMeetLogStorage {
         for (let i = 0, len = logEntries.length; i < len; i++) {
             const logEntry = logEntries[i];
 
-            if (typeof logEntry === 'object') {
-                // Aggregated message
-                logMessage += `(${logEntry.count}) ${logEntry.text}\n`;
-            } else {
-                // Regular message
-                logMessage += `${logEntry}\n`;
+            if (logEntry.timestamp) {
```

Comment:
What happens if logEntry is undefined?

Reply:
it is never supposed to be undefined, the app will crash

Reply:
I think that even if log method is called empty, then the logger lib will still produce the logEntry object with a timestamp but no text. I will check if that cases works. I can't think of any other case that could lead to problems here.

Reply:
Yeah just tested the empty case. I don't think this should ever crash unless the logger library is broken which should be detected on the dev stage.